packages/control-plane/src/auth/openssh-ed25519.ts

```
@@ -0,0 +1,220 @@
+  if (publicKey.length !== ED25519_PUBLIC_KEY_BYTES || reader.remaining !== 0) {
+    throw new OpenSshKeyValidationError("Invalid Ed25519 public key");
+  }
+  return publicKey;
+}
+
+function parsePrivateKeyBlob(privateKeyBlob: Uint8Array): {
+  publicKey: Uint8Array;
+  seed: Uint8Array;
+} {
+  const reader = new BinaryReader(privateKeyBlob);
+  if (reader.readUint32() !== reader.readUint32()) {
```

Biome (2.5.3)

[error] This comparison uses the same expression on both sides.

(lint/suspicious/noSelfCompare)